integration/tests/basic/rpc/test_rpc_base_calls.py

```diff
@@ -0,0 +1,650 @@
+]
+
+
+def get_event_signatures(abi: tp.List[tp.Dict]) -> tp.List[str]:
+    """Get topics as keccak256 from abi Events"""
+    topics = []
+    for event in filter(lambda item: item["type"] == "event", abi):
+        input_types = ",".join(i["type"] for i in event["inputs"])
+        signature = f"{event['name']}({input_types})"
+        topics.append(f"0x{keccak(signature.encode()).hex()}")
+    return topics
+
+
+@allure.feature("JSON-RPC validation")
+@allure.story("Verify JSON-RPC proxy calls work")
+class TestRpcBaseCalls(BaseMixin):
+    _erc20_contract: tp.Optional[tp.Any] = None
+
+    @pytest.fixture
+    def erc20_contract(self) -> tp.Any:
+        if not TestRpcBaseCalls._erc20_contract:
+            contract, contract_deploy_tx = self.web3_client.deploy_and_get_contract(
+                "ERC20/ERC20.sol",
+                "0.8.8",
+                self.sender_account,
+                contract_name="ERC20",
+                constructor_args=["Test Token", "TT", 1000],
+            )
+
+            tx_receipt = self.web3_client.send_erc20(
+                self.sender_account,
+                self.recipient_account.address,
+                amount=1,
+                address=contract_deploy_tx["contractAddress"],
+                abi=contract.abi,
+            )
+            self.wait_transaction_accepted(tx_receipt.transactionHash.hex())
+            TestRpcBaseCalls._erc20_contract = contract, contract_deploy_tx, tx_receipt
+        return TestRpcBaseCalls._erc20_contract
+
+    def test_eth_call_without_params(self):
+        """Verify implemented rpc calls work eth_call without params"""
+        response = self.proxy_api.send_rpc("eth_call")
+        assert "error" in response, "Error not in response"
+
+    @pytest.mark.parametrize("tag", [Tag.LATEST, Tag.PENDING, Tag.EARLIEST])
+    def test_eth_call(self, tag):
+        """Verify implemented rpc calls work eth_call"""
+        params = [
+            {"to": self.recipient_account.address, "data": hex(pow(10, 14))},
+            tag.value,
+        ]
+
+        response = self.proxy_api.send_rpc("eth_call", params=params)
+        assert "error" not in response
+        assert (
+                response["result"] == "0x"
+        ), f"Invalid response result, `{response['result']}`"
+
+    def test_eth_get_transaction_receipt_with_incorrect_hash(self):
+        """Verify implemented rpc calls work eth_getTransactionReceipt when transaction hash is not correct"""
+
+        response = self.proxy_api.send_rpc(
+            method="eth_getTransactionReceipt", params=gen_hash_of_block(31)
+        )
+        assert "error" in response
+        assert response["error"]["message"] == "transaction-id is not hex"
+
+    def test_eth_gas_price(self):
+        """Verify implemented rpc calls work eth_gasPrice"""
+        response = self.proxy_api.send_rpc("eth_gasPrice")
+        assert "error" not in response
+        assert rpc_checks.is_hex(
+            response["result"]
```

Comment:
add check that this value more than 100000000 because right now gas price 911479586960

Reply:
Good point, added